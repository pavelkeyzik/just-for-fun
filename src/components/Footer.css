.Footer {
  border-top: 1px solid #333;
  color: #aaa;
  padding: 20px;
}
