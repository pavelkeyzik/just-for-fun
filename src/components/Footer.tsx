import React from 'react';

import styles from './Footer.module.css';

export function Footer(): JSX.Element {
  return <div className={styles.footer}>Make fun with love</div>;
}
