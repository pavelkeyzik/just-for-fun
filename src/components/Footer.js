import React from 'react';

import './Footer.css';

export function Footer() {
  return <div className="Footer">Make fun with love</div>;
}
