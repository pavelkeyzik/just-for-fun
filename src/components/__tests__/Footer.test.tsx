import React from 'react';
import { render } from '@testing-library/react';
import { Footer } from '../Footer';

test('Check snapshot', () => {
  const { container } = render(<Footer />);

  expect(container.firstChild).toMatchSnapshot();
});

test('Check text in footer', () => {
  const { container } = render(<Footer />);

  expect(container.textContent).toEqual('Make fun with love');
});
