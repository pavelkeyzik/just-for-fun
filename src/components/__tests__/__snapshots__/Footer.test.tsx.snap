// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check snapshot 1`] = `
<div
  class="footer"
>
  Make fun with love
</div>
`;
